packages/server/duckdb-server-go/go.mod

```diff
@@ -0,0 +1,40 @@
+module github.com/uwdata/mosaic/packages/server/duckdb-server-go
+
+go 1.24.4
+
+require (
+	github.com/apache/arrow-go/v18 v18.3.1
+	github.com/coder/websocket v1.8.13
+	github.com/marcboeker/go-duckdb/v2 v2.3.2
+	github.com/maypok86/otter/v2 v2.0.0
+	github.com/stretchr/testify v1.10.0
+)
+
+require (
```

Comment:
Not super familiar with go development but do we need to specify the indirect dependencies?

Reply:
Yes, this is a very standard, cleaned (`go mod tidy`) file. I tried to keep the imports as lean as possible.